build-aux/snap/snapcraft.yaml

```diff
@@ -206,12 +206,19 @@ parts:
       if [ "${VERSION/+fips/}" != "$VERSION" ] ; then
           # we have a '+fips' element in the version, which may be coming from
           # debian/changelog or git tag
-          echo "-- detected FIPS build"
-          touch fips-build
+          echo "-- detected FIPS build from version $VERSION"
+          touch "${CRAFT_PROJECT_DIR}/fips-build"
       fi
-      # TODO detect when doing a FIPS snap build on LP
 
-      if [ -f fips-build ]; then
+      # check for triggers that indicate a build on LP of the snapd-fips snap
+      # recipe https://launchpad.net/~fips-cc-stig/fips-cc-stig/+snap/snapd-fips
+      if [ "$(./release-tools/is-lp-fips-build.sh)" = "true" ]; then
+          echo "-- detected FIPS build from LP location"
+          touch "${CRAFT_PROJECT_DIR}/fips-build"
```

Comment:
It might be more clear to just have one file, `fisp-build`, and then have the value of that file indicate the type of build happening.